Add option to show piece notation to move list in analysis view

freefal opened the pull request:
Closes https://github.com/veloce/lichobile/issues/412

veloce commented:
Thanks for that!

It would be even nicer if this was a global settings, under 'settings -> game display' entry probably, and would display the piece notation also in game replay (visible in landscape mode)

freefal commented:
It is in 'settings -> game display'. I'll make game replay use it as well.
I never go into landscape so didn't know it was used elsewhere.

On Tue, Oct 11, 2016 at 3:38 AM, Vincent Velociter <notifications@github.com
> wrote:

> Thanks for that!
>
> It would be even nicer if this was a global settings, under 'settings ->
> game display' entry probably, and would display the piece notation also in
> game replay (visible in landscape mode)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/veloce/lichobile/pull/446#issuecomment-252835973>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AA6Rdg8zzkM9qMYt2IGSV4EFynWfGwIEks5qyzz_gaJpZM4KTQj0>
> .
>

veloce commented:
One more location to change: offline games in landscape mode.

veloce commented on project/src/js/ui/shared/round/view/replay.js:
```diff
@@ -1,4 +1,5 @@
 import * as helper from '../../../helper';
+import settings from '../../../settings';
```
bad path

veloce commented:
You could cache access to the pref since it's been read a lot during redraws and it's localstorage.

See how I do it in `Board.tsx` for pieceTheme and boardTheme as an example.

freefal commented:
Apologies for the bad settings. I thought I tested everywhere I added the
pieceNotation, but clearly I didn't. Now fixed.
Thanks for the tip on caching. Now implemented.

On Sat, Oct 15, 2016 at 1:11 PM, Vincent Velociter <notifications@github.com
> wrote:

> You could cache access to the pref since it's been read a lot during
> redraws and it's localstorage.
>
> See how I do it in Board.tsx for pieceTheme and boardTheme as an example.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/veloce/lichobile/pull/446#issuecomment-253997281>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AA6RdlzXX3J1BibWylZDvNR50JCOA9KTks5q0Qk9gaJpZM4KTQj0>
> .
>
